### DISCUSSION TITLE
Enable ordering by runtime in Actions view

### DISCUSSION THREAD

**Discussion by krystofspiller** [community/community#14364] [Actions]
Currently, it is not possible to change the default ordering of items in Actions view, i.e. github.com/&lt;organization&gt;/&lt;repo&gt;/actions. It orders the items by when they were run. It would be useful to be able to order items by their runtime as well. In this way, it would be easy to see anomalous runs, both long and short.

I also imagine that in such a use case it would also be useful to filter actions from only a recent time period (1 week, 1 month, etc.) as opposed to all of them.

**Comment by ethomson** [comment]
Thanks for the feedback @Laegas. I agree that it would be nice to have more insight on this page. I'll make sure that we consider this feedback when we make improvements here.
